### PR TITLE
VACMS-6935 - Create metatag audit view

### PR DESCRIPTION
**Pull request by msbtterswrth**
## Description
Created a view to help audit the current meta tags and how they're being used across all Veteran-facing content types.

Relates to #6935.

## Testing done
Navigated the view and saw the various fields being displayed for meta tags within the view results list.

## Screenshots
image

## QA steps

As cms editor I can
1. View the metatag audit tool (/admin/content-tools/metatag-audit) and
1. View a list of content and the meta data stored in the CMS
1. Then validate Acceptance Criteria from issue
- [x] See what metadata drupal currently has (and what it has to offer that isn’t currently used)

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [ ] `Sitewide CMS Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/views.view.metatag_audit.yml [hunk 1]
```diff
@@ -0,0 +1,1472 @@
...
+        - 'config:field.storage.node.field_meta_title'
+        - 'config:workflow_list'
+  metatag_audit:
+    display_plugin: page
+    id: metatag_audit
+    display_title: 'Metatag Audit'
+    position: 1
+    display_options:
+      display_extenders: {  }
+      path: admin/content-tools/metatag-audit
```
Comment (swirtSJW) [inline config/sync/views.view.metatag_audit.yml:1452]: This path deviates from our existing audit related views of `/content/audit/*`
```suggestion
      path: admin/content/audit/metatags
```
image

**Review by swirtSJW — CHANGES_REQUESTED** [review]
Looks good overall.   Just some standardization things
image
- [ ] "Edit" should be  button.
- [ ]  Date display should match existing patttern

**Comment by swirtSJW** [comment]
A couple other issues surfaced.
- [ ] The section filter is not working properly 
image
This should not be the field_administration filter it should be "workbench access section".  When you select a VISN, you should get all sections within that VISN.


- [ ] The filters need a "Reset" button 
- [ ] The "Apply" button should say "Filter"  
image

**Review by swirtSJW — CHANGES_REQUESTED** [review]
I am still seeing this issue 
- [x] The section filter is not working properly
image

This should not be the field_administration filter it should be "workbench access section". When you select a VISN, you should get all sections within that VISN.

**Comment by swirtSJW** [comment]
Also something in your git process is still creating these repeating commits.
image

Please make sure to run the git config steps here https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/READMES/getting-started.md

**Comment by msbtterswrth** [comment]
@swirtSJW i did update it to use workbench access, ass seen here: 
image

I am not sure what else I need to do? 

As far as the git commits go, I did follow the git readme when i set it up originally, and again reset it it up last week. these were all different commits updating the tests, though I thought i had squashed the two yesterday into one commit, but I guess they still came through separately. 

I will remind myself not to reuse the same commit message if that's helpful.

**Review by swirtSJW — APPROVED** [review]
Tests out good.  Nice work.
image

**Pull request merged**